### ISSUE TITLE
Container and View Component

### ISSUE THREAD

**Issue by vanhtuan0409** [facebook/flux#328]
http://facebook.github.io/flux/docs/flux-utils.html#content
Flux Document said that Container should have no props but in some cases I find it quite hard to follow this principle. For example I have 3 component:

- Post list
  - Post
    - Comment list

The ``Post list`` can be easily wrap with container fetch data from store
But I need ``Post`` id to fetch ``Comment list``
So my question is if I want to create a Container for ``Comment list``, should I pass in some props or how to keep the Container pure

**Comment by kyldvs** [comment]
Yeah this is a case where it's okay to pass props to the container. In particular passing an ID to a container is okay when it's needed, I haven't really run into any other valid use cases though
